### DISCUSSION TITLE
Is it possible to let users in LAN access the SDXL installed locally?

### DISCUSSION THREAD

**Discussion by kanarch66** [vladmandic/sdnext#1624] [Q&A] [answered]
SD1.5 I used to use --listen
For this version, is it possible to use --listen? If so, where to put this command at?
Thank you very much for your answers everyone.

**Comment by vladmandic** [comment]
yes, nothing changed in that aspect.

**Reply by kanarch66** [reply, marked as answer]
Thank you for your answer.  where should I put --listen? Please point to me the file and the line in the file.  I am so new to this thing.

**Reply by kanarch66** [reply]
Thank you. I found it :)

**Comment by kanarch66** [comment]
Just make another bat file and put --listen in the same line.
